public/main.js

```diff
@@ -117,4 +124,21 @@ function deleteTodo(id, callback) {
     };
 }
 
+function deleteCompleted() {
+    var completed = [];
+    getTodoList(function(todos) {
+        todos.forEach(function (todo) {
+            if (todo.isComplete) {
+                console.log(todo.id);
+                completed.push(todo.id);
+            }
+        });
+        console.log(completed);
+        completed.forEach(function (id) {
+            deleteTodo(id, function () {});
```

Comment:
I'm potentially deleting a lot of items so if each delete is successful I don't want to do anything about it until all of the deletes are done, when the list of items will refresh.